collectors/bzimport/tests/test_convertors.py

```diff
@@ -258,16 +269,16 @@ def test_basic(self):
         reference = FlawReference.objects.first()
         cvss_score = FlawCVSS.objects.first()
         tracker = Tracker.objects.first()
-        package_versions = CVEv5PackageVersions.objects.first()
-        version = CVEv5Version.objects.first()
+        package_versions = Package.objects.first()
+        version = PackageVer.objects.first()
```

Comment:
Just a recommendation, but I would use better names, e.g. `package` for `Package` and `package_version` for `PackageVer`. The old names still implies the old models.

Reply:
Fixed in commit [0986a35](https://github.com/RedHatProductSecurity/osidb/pull/307/commits/0986a3582c8c4dba8c830bdf8173d2f6f7b98c16)